README.md

```diff
@@ -0,0 +1,75 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
+![screenshot]()
+
+>
+
+## Live Demo
+
+[Live Demo Link]()
```

Comment:
- [ ] **[OPTIONAL]** 
- [ ] You can decide to remove this empty screenshot and live demo since it is not available yet so that your README looks more professional.